.npmrc

```diff
@@ -1 +1,2 @@
 engine-strict=true
+legacy-peer-deps=true
```

Comment:
Needed for Storybook when running npm 8. I think they are fixing this in Storybook 7.

Reply:
this could be part of why we had a peer deps issue in sdk? random aside.

Reply:
Was thinking that too.